tests/run_pretrained_models.py

```diff
@@ -206,11 +206,10 @@ def run_onnxmsrt(self, name, onnx_graph, inputs):
             self.onnx_runtime = time.time() - start
         return results
 
-    def run_onnxmsrtnext(self, name, onnx_graph, inputs):
+    def run_onnxmsrtnext(self, name, model_proto, inputs):
         """Run test against msrt-next backend."""
```

Comment:
when running this with your changes:
mobilenet_v1_75_192
        downloaded /tmp/pre-trained\mobilenet_v1_0.75_192_frozen\mobilenet_v1_0.75_192/frozen_graph.pb
2018-08-22 14:28:57.717831: I T:\src\github\tensorflow\tensorflow\tools\graph_transforms\transform_graph.cc:318] Applying fold_constants
2018-08-22 14:28:57.870910: I T:\src\github\tensorflow\tensorflow\tools\graph_transforms\transform_graph.cc:318] Applying fold_batch_norms
2018-08-22 14:28:58.012064: I T:\src\github\tensorflow\tensorflow\tools\graph_transforms\transform_graph.cc:318] Applying fold_old_batch_norms
        tensorflow OK
before optimization: ops statistics: Counter({'Transpose': 58, 'Conv': 28, 'Add': 28, 'Max': 27, 'Min': 27, 'Mul': 13, 'Identity': 1, 'Softmax': 1, 'Reshape': 1, 'AveragePool': 1, 'Squeeze': 1})
        to_onnx FAIL name 'TensorShapeProto' is not defined
        run_onnx FAIL 'NoneType' object has no attribute 'SerializeToString'

mobilenet_v1_100_224
        downloaded /tmp/pre-trained\mobilenet_v1_1.0_224_frozen\mobilenet_v1_1.0_224/frozen_graph.pb
2018-08-22 14:28:15.857182: I T:\src\github\tensorflow\tensorflow\tools\graph_transforms\transform_graph.cc:318] Applying fold_constants
2018-08-22 14:28:16.066649: I T:\src\github\tensorflow\tensorflow\tools\graph_transforms\transform_graph.cc:318] Applying fold_batch_norms
2018-08-22 14:28:16.295635: I T:\src\github\tensorflow\tensorflow\tools\graph_transforms\transform_graph.cc:318] Applying fold_old_batch_norms
        tensorflow OK
before optimization: ops statistics: Counter({'Transpose': 58, 'Conv': 28, 'Add': 28, 'Max': 27, 'Min': 27, 'Mul': 13, 'Identity': 1, 'Softmax': 1, 'Reshape': 1, 'AveragePool': 1, 'Squeeze': 1})
        to_onnx FAIL name 'TensorShapeProto' is not defined
        run_onnx FAIL 'NoneType' object has no attribute 'SerializeToString'

Reply:
oh, my bad, it's a last minute change. I'll fix it. sorry.